plugins/package-managers/conan/src/main/kotlin/ConanV1Handler.kt

```diff
@@ -56,7 +56,8 @@ internal class ConanV1Handler(private val conan: Conan) : ConanVersionHandler {
                 workingDir,
                 "info", definitionFile.name,
                 "-l", lockfileName,
-                "--json", jsonFile.absolutePath
+                "--json", jsonFile.absolutePath,
+                *DUMMY_COMPILER_SETTINGS
```

Comment:
I'm not sure about this. Didn't @heliocastro have some problem with passing these unconditionally?

Reply:
@heliocastro had this PR that he unfortunately gave up on:
https://github.com/oss-review-toolkit/ort/pull/10127/commits

He wanted to remove the dummy parameters altogether  and rely on an autogenerated profile instead.

However, note that this change in current PR only applies if a _lockfile_ is given and is merely an alignment: why, in 1 case out of 4 (Conan 1/2, lockfile/no lockfile) should them not be passed ?

Reply:
> That basic idea still makes sense to me. Maybe this could be retried without doing any other changes that are not strictly required in this context?

I agree. But could I rather do it in a different PR, as I am limited in my dev time till Thursday ?

Reply:
> But could I rather do it in a different PR, as I am limited in my dev time till Thursday ?

Of course. I was not meaning to imply doing it now.

Reply:
> why, in 1 case out of 4 (Conan 1/2, lockfile/no lockfile) should them not be passed ?

I guess [this](https://github.com/oss-review-toolkit/ort/actions/runs/19425765675/job/55575017248?pr=11091#step:7:2517) "Cannot use profile, settings, options, env or conf 'host' when using lockfile" error we now see in CI was the reason for not adding it here.

Mind adding a code comment for that to remind us?